### DISCUSSION TITLE
Welcome to prodict Discussions!

### DISCUSSION THREAD

**Discussion by ramazanpolat** [ramazanpolat/prodict#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

I'll try to do my best to be involved in discussions.

**Comment by FICU-tensileai** [comment]
Has anyone run into any problems getting this to be recognized within VS Code? Ran the pip install from VS Terminal and restarted but it still doesn't recognize the library in the path. Not sure if it's a prodict thing or a VS Code issue.

**Reply by gosuto-inzasheru** [reply]
same here, not getting suggestions in vscode editor.

**Reply by ramazanpolat** [reply]
Just installed a fresh version of VS Code and checked. It works for me:

![image](https://user-images.githubusercontent.com/3527695/148584242-d44ce2d6-2368-45d7-bea1-fb1b2980fa08.png)
